alica_tests/src/test/test_assignment.cpp

```diff
@@ -45,8 +41,12 @@ TEST(Assignment, RobotsInserted)
     ASSERT_TRUE(*robot1 > *robot2);
     ASSERT_TRUE(*robot1 < *robot3);
 
+    alica::AlicaContext *ac = new alica::AlicaContext("Roleset", "MasterPlan", true, path + "/etc/Alica_nase.yaml");
+    ac->setLocalAgentName("nase");
+
     PlanRepository repo;
-    ModelManager modelManager(repo);
+    alica::AlicaEngine *ae = alica::AlicaTestsEngineGetter::getEngine(ac);
```

Comment:
umm passing alica engine came with this side effect. Every module would need engine to be unit tested. Let's figure out a strategy and fix this in a different PR.